api/workAllocation/caseWorkerService.ts

```diff
@@ -47,3 +48,32 @@ export async function handlePostSearch(path: string, payload: string | any, req:
     const response: AxiosResponse = await http.post(path, payload, { headers })
     return response
 }
+
+export async function handlePostRoleAssingnments(path: string, payload: any, req: EnhancedRequest): Promise<any> {
+    const headers = setHeaders(req)
+    headers.pageNumber = 0
+    headers.size = MAX_RECORDS
+    // sort
+    // direction
+    const response: AxiosResponse = await http.post(path, payload, { headers })
+    return response
+}
+
+export async function handlePostCaseWorkersRefData(path: string, userIds: any, req: EnhancedRequest): Promise<any> {
+    const payload = {
+        userIds,
+    }
+    const headers = setHeaders(req)
+    const response: AxiosResponse = await http.post(path, payload, { headers })
+    return response
+}
+
+export function getUserIdsFromRoleApiResponse(response: any): string [] {
+    let userIds = new Array<string>()
+    if (response && response.roleAssignmentResponse) {
+        response.roleAssignmentResponse.forEach(roleAssingment => {
```

Comment:
Spelling of roleAssignment (roleAssingment) slightly off here. Should be quick change